frontend/pages/Homepage/cards/WelcomeHost/WelcomeHost.tsx

```diff
@@ -246,18 +246,15 @@ const WelcomeHost = ({
             if (p.response) {
               return (
                 <Button
-                  variant="text-icon"
+                  variant="unstyled"
                   onClick={() => handlePolicyModal(p.id)}
                 >
                   <div className="policy-block">
-                    <div className="info">
-                      <img
-                        alt={p.response}
-                        src={p.response === policyPass ? IconPassed : IconError}
-                      />
-                      {p.name}
-                    </div>
-                    <img alt="" src={IconChevron} />
+                    <img
+                      alt={p.response}
+                      src={p.response === policyPass ? IconPassed : IconError}
```

Comment:
Stylistic nit but I'd probably do a quick rename in the file to use the constant case here on `POLICY_PASS` and `POLICY_FAIL`.